src/constants.ts

```diff
@@ -7,9 +7,9 @@ export const PROGRAM_VERSION = packageDotJson.version;
 export const USER_AGENT = `CrossSeed/${PROGRAM_VERSION}`;
 
 export const EP_REGEX =
-	/^(?<title>.+?)[\s._](?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)/i;
+	/^(?<title>.+?)[\s._](?=(?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)|(?<date>\d{4}[\s._-]\d{2}[\s._-]\d{2}))/i;
```

Comment:
This bugfix, and further looking into searching for daily shows, effectively changes how I would address your EP_REGEX.

You will need to capture the date's numerals independently if we are going to integrate searching for them effectively. TV search capable trackers will have the season be <year> and the episode be <month>/<day> from what I can see in prowlarr.

**EP_REGEX** (https://regex101.com/r/fuE0bM/3) 
```suggestion 
	/^(?<title>.+?)[\s._-]+(?:(?<season>S\d+)?[_.\s]?(?<episode>E\d+(?:[-\s]?E?\d+)?)|(?<date>(?<year>\d{4})[\s._-](?<month>\d{2})[\s._-](?<day>\d{2})))/i;`